experiments/tester/src/RNTester/TestComponents/PersonaCoin/CustomizeUsage.tsx

```diff
@@ -1,76 +1,19 @@
 import * as React from 'react';

```

Comment:
Ignore this file. I have sent out a separate PR for this change. Including it here to pass the CI build.